web/scripts/rate_all.py

```diff
@@ -100,8 +99,10 @@ def rate_all(t_from=None, t_to=None, iters=200, lam=.22):
             for k,v in rating_prior.items():
```

Comment:
re: outliers  it's certainly possible.  I didn't test any crazy outliers on the bottom side -- on the top side, it seems to do 'the right thing' on the AGA's set, which has some undefeated players on the top like Rui Naiwei, Myungwan Kim, etc.

re: renormalization:  I'm not sure.  I added this when i was still having some non-convergence issues, but it might not still be needed.